### ISSUE TITLE
Language "" cannot be installed, Installation folder cannot be deleted.

### ISSUE THREAD

**Issue by coolcat-creations** [joomla/joomla-cms#39139]
### Steps to reproduce the issue
I had 10 subdomains on the exact same environment, same time, same devices,same Joomla version (4.2.4)
When it came to install additional languages (German) I got an error: unable to install language "" on two of the sites. Both sites also the installation folder could not be deleted 

### System information (as much as possible)
Php 8
SSL
Joomla 4.2.4

**Comment by brianteeman** [comment]
There is either a file permissions error on those two sites or possibly even your hosting has reached its maximum space.

This really is a support question and therefore should be on the forum

**Comment by coolcat-creations** [comment]
No, it did not reach the maximum space and it happened in summer as well when everyone was on their local environment. This time to prevent it I prepared a server for it so we are not limited by wrong local setup. It was said that this particular bug was fixed but seems it's not.

**Comment by coolcat-creations** [comment]
@richard67 and @Hackwar you said im August this error was fixed for Joomla 4.2 ?

**Comment by richard67** [comment]
There was this fix for the problem when deleting the installation folders had failed: #37938 .

**Comment by coolcat-creations** [comment]
<img width="519" alt="grafik" src="https://user-images.githubusercontent.com/828371/199739925-ece17873-7842-4150-a513-d087e166e7a6.png">


PHP Built On | Linux dd25412 4.15.0-194-generic #205-Ubuntu SMP Fri Sep 16 19:49:27 UTC 2022 x86_64
-- | --
mysql
5.7.37-nmm1-log
utf8mb4_general_ci
utf8mb4_general_ci
None
Yes
8.0.20
Apache
fpm-fcgi
Joomla! 4.2.4 Stable [ Uaminifu ] 25-October-2022 15:20 GMT
Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:106.0) Gecko/20100101 Firefox/106.0

**Comment by wojtekxtx** [comment]
> There was this fix for the problem when deleting the installation folders had failed: #37938 .

@richard67 nice fix, if only it worked..................

> There is either a file permissions error on those two sites or possibly even your hosting has reached its maximum space.

@brianteeman Judging by what OP wrote, non e of the above is true. And where better to get support for **code issues** than on official project's issues page within repo? OP was correct by placing it here ( where it belongs ).

@coolcat-creations  Your version  of J! seems not to be compatible with your `php` version. As a sidenote: Joomla have some weird issues with `8.x` branch of `php`.

**Comment by Quy** [comment]
@coolcat-creations Please advise status. Thanks.

**Comment by coolcat-creations** [comment]
@Quy What Status would you need excatly? Only thing I can check is to see at my next class in summer if the issue is still the same. Hoped someone has a clue where the error
 unable to install language "" 
 could come from or how to rewrite the error message to make it more clear.

**Comment by Hackwar** [comment]
Can you check this with Joomla 5.1 again? Joomla 5.1 has some bugfixes which are not in 5.0 (and as of yet not yet in 5.2) and fixed some issues with installing languages.

**Comment by brianteeman** [comment]
> Installation folder cannot be deleted

this is still an issue on windows servers.

**Comment by Quy** [comment]
> > Installation folder cannot be deleted
> 
> this is still an issue on windows servers.

https://github.com/joomla/joomla-cms/issues/37909

**Comment by coolcat-creations** [comment]
I cant check because it was on a specific environment I don't have access to anymore ;(

**Comment by brianteeman** [comment]
No point in keeping it open then

**Comment by coolcat-creations** [comment]
The issue was mentioned in other issue as a duplicate so maybe its still a problem? I would not close an issue that hold another issue?

**Comment by chmst** [comment]
The error with deleting the installation folder on windows still exists, win11, xampp, php 8.2

**Comment by coolcat-creations** [comment]
But if we close this we need to reopen this: https://github.com/joomla/joomla-cms/issues/42295 or not?
